app/views/admin/SkippedContactsView.coffee

```diff
@@ -1,8 +1,32 @@
-RootComponent = require 'views/core/RootComponent'
```

Comment:
I tried not to include too much user-facing stuff in this since this is just an architectural change, but I can convert the view that uses RootComponent (ProjectGalleryView) and include it in this if you'd like.